### PR TITLE
[Fix] dep : Updated & in-sync the dependencies with chaos-operator

### PR DESCRIPTION
**Pull request by rahulchheda**
**What this PR does / why we need it**:
- After fetching the litmuschaos/chaos-operator:master, building the chaos-executor, will not work until the chaos-operator k8s dep, are updated.
- Made the dependencies in-sync with the chaos-operator
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -29,33 +29,33 @@
...
-  name = "sigs.k8s.io/controller-runtime"
-  version = "=v0.1.10"
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.15.4"
 
 [[override]]
-  name = "github.com/litmuschaos/chaos-operator"
-  branch = "master"
-  
+  name = "github.com/coreos/prometheus-operator"
```
Comment (ksatchit) [inline Gopkg.toml]: are we really using this somehere in executor?

File: Gopkg.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+[[constraint]]
+  name = "github.com/Sirupsen/logrus"
+  version = "1.4.1"
+
+[[constraint]]
+  name = "github.com/prometheus/client_golang"
+  version = "0.9.2"
+
+[[constraint]]
```
Comment (Sanjay1611) [inline Gopkg.toml]: I think we are using `kubernetes-1.15.4` in chaos-operator

Comment (rahulchheda) [reply Gopkg.toml]: Please check once again, it is 1.15.4

File: Gopkg.toml (hunk 1 above)
Comment (rahulchheda) [reply Gopkg.toml]: Removing it!

**Pull request merged**